### PR TITLE
Some Improvement Test Coverage of InterceptorTest

### PR DESCRIPTION
**Pull request by iNoles**
This will increase some coverage on FuelManager.

Only thing is missing that keystore for socketFactory isn't being tested.
One partial hit from download for parameters.

### CODE REVIEW

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt [hunk 1]
```diff
@@ -612,4 +614,19 @@ class InterceptorTest : BaseTestCase() {
...
         assertThat(error, notNullValue())
     }
+
+    @Test
+    fun testRemoveAllRequestInterceptors() {
+        val manager = FuelManager()
+        val (request, response, _) = manager.request(Method.GET,
+                "http://httpbin.org/status/418").response()
+
+        //val (data, error) = result
```
Comment (kittinunf) [inline fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt]: Uncomment?

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt [hunk 2]
```diff
@@ -612,4 +614,19 @@ class InterceptorTest : BaseTestCase() {
...
+        assertThat(request, notNullValue())
+        assertThat(response, notNullValue())
+        //assertThat(error, nullValue())
```
Comment (kittinunf) [inline fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt]: Here too?

File: fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt (hunk 1 above)
Comment (iNoles) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt]: There is a reason that I didn't comment these, those are null.

Comment (iNoles) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt]: I could remove these comments if you want.

Comment (kittinunf) [reply fuel/src/test/kotlin/com/github/kittinunf/fuel/InterceptorTest.kt]: Hmm do you know why though? If that's the case, we might not be able to properly test the effect of calling `removeAllRequestInterceptors `

**Comment by kittinunf** [comment]
Merge this!

**Pull request merged**